ansible/roles/mycar/files/custom/car/services/jobs/job_record.py

```diff
@@ -153,7 +153,7 @@ def on_training_job_update(self, event_dict: dict):
             self.display_screen_msg(f'Annulation de l\'entrainement (j{event.job.job_id}) ⏳...')
         elif event.job.state == JobState.RUNNING:
             self.logger.debug('Job[job_id: %i] Training job (id: %i) is running \\o/', self.get_id(), event.job.job_id)
-            self.display_screen_msg("Entrainement du modèle en cours 🎓⏳️️... ")
+            self.display_screen_msg("UI-train | Entrainement du modèle en cours 🎓⏳️️... ")
```

Comment:
I don't understand why `UI-train` it's still here ? Didn't we remove it so that we could still use the old front in case of any failure.
At Bordeaux it did work without this `UI-train`, I think it should be removed.

Reply:
yes my bad, we didn't change the ansible